profiling/energy-profiler/common/timer_system.cpp

```diff
@@ -0,0 +1,318 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include "timer_system.hpp"
+
+#include <cstdio>
+#include <iostream>
+
+namespace KokkosTools {
+namespace Timer {
```

Comment:
hmm `Timer` might be a bit of a general name and raise the question why this is repeated from the other timer